backend/functions/lib/adapters/controllers/case.assignment.controller.test.ts

```diff
@@ -0,0 +1,207 @@
+import { CaseAssignmentRole } from '../types/case.assignment.role';
+import { CaseAssignmentController } from './case.assignment.controller';
+import { CaseAssignmentRepositoryInterface } from '../../interfaces/case.assignment.repository.interface';
+import { CaseAssignmentLocalRepository } from '../gateways/case.assignment.local.repository';
+import { TrialAttorneysAssignmentRequest } from '../types/trial.attorneys.assignment.request';
+import { AttorneyAssignmentResponseInterface } from '../types/case.assignment';
+import { httpError } from '../utils/http';
+import { AssignmentException } from '../../use-cases/assignment.exception';
+const context = require('azure-function-context-mock');
+describe('Chapter 15 Case Assignment Creation Tests', () => {
+  test('A chapter 15 case is assigned to an attorney when requested', async () => {
+    const testCaseAssignment = new TrialAttorneysAssignmentRequest(
+      '12345',
+      ['8082'],
+      CaseAssignmentRole.TrialAttorney,
+    );
+
+    const mockCaseAssignmentRepository: CaseAssignmentRepositoryInterface =
+      new CaseAssignmentLocalRepository();
```

Comment:
These mock repositories are not actually mocks but real instances of the local repository.

Reply:
Renamed to caseAssignmentLocalRepository